job-dsls/jobs/prod/prod_bamoe_kogito_properties_generator.groovy

```diff
@@ -46,8 +46,8 @@ pipelineJob("${folderPath}/bamoe-kogito-properties-generator") {
     description("Generate properties files for BAMOE Kogito productized builds")
 
     parameters {
-        stringParam("REPO_URL", "http://download.devel.redhat.com/rcm-guest/staging")
-        stringParam("DELIVERABLE_REPO_URL", "http://download.devel.redhat.com/devel/candidates/middleware")
+        stringParam("REPO_URL", "\${env.STAGING_SERVER_URL}")
```

Comment:
I think using `env` here will break as it is a Jenkins parameters. Please remove `env` here and all in other places that is a parameter.

Reply:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.